src/sentry/integrations/slack/notifications.py

```diff
@@ -52,7 +55,7 @@ def get_attachments(
 
 
 def get_context(
-    notification: OrganizationNotification,
+    notification: BaseNotification,
```

Comment:
A Slack integration is always related to an organization, so why is this changing?